src/globals.py

```diff
@@ -12,6 +12,3 @@ def init() -> None:
     except ImportError:
         print('Warning: Tkinter is not installed, graphs will not be shown')
         CAN_SHOW_GRAPH = False
-
-    HOURS_LIST = ['00', '01', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11',
```

Comment:
Why did you remove this?

Reply:
It came in as a linting error in flake8 as it wasn't being used in the program.